tools/wasme/cli/docs/content/reference/operator/filter_deployment.md

```diff
@@ -1,236 +0,0 @@
-
```

Comment:
I don't think this should be deleted, it is the source for some of our reference docs, eg - https://docs.solo.io/web-assembly-hub/latest/reference/operator/filter_deployment/

I am however able to reproduce this change locally from codegen. I will investigate and get back to you on this piece.